output_template/python/lmnet/utils/image.py

```diff
@@ -0,0 +1 @@
+lmnet/lmnet/utils/image.py
```

Comment:
I think this path should be relative path from  `output_template/python/lmnet/utils/image.py`, Maybe `../../../../lmnet/lmnet/utils/image.py` is correct path 🤔

Reply:
@ruimashita Thanks! Fixed.